packages/fast-components-react-msft/src/action-trigger/action-trigger.spec.tsx

```diff
@@ -0,0 +1,209 @@
+import * as React from "react";
+import * as ShallowRenderer from "react-test-renderer/shallow";
+import * as Adapter from "enzyme-adapter-react-16/build";
+import { configure, mount, shallow } from "enzyme";
+import examples from "./examples.data";
+import MSFTActionTrigger, {
+    ActionTriggerAppearance,
+    ActionTriggerHandledProps,
+    ActionTriggerManagedClasses,
+    ActionTriggerProps,
+    ActionTriggerUnhandledProps,
+} from "./action-trigger";
+import { ActionTrigger } from "./index";
+import { ActionTriggerClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/*
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+describe("action trigger", (): void => {
+    const managedClasses: ActionTriggerClassNameContract = {
+        actionTrigger: "action-trigger",
+        actionTrigger_glyph: "glyph",
+        actionTrigger__primary: "action-trigger-primary",
+        actionTrigger__outline: "action-trigger-outline",
+        actionTrigger__lightweight: "action-trigger-lightweight",
+        actionTrigger__justified: "action-trigger-justified",
+        actionTrigger__disabled: "action-trigger-disabled",
+    };
+    const href: string = "#";
+
+    test("should have a displayName that matches the component name", () => {
+        expect((MSFTActionTrigger as any).name).toBe(MSFTActionTrigger.displayName);
+    });
+
+    test("should not throw if managedClasses are not provided", () => {
+        expect(() => {
+            shallow(
+                <MSFTActionTrigger
+                    /* tslint:disable-next-line */
+                    glyph={(className?: string): React.ReactNode => {
+                        return <div>X</div>;
+                    }}
+                />
+            );
+        }).not.toThrow();
+    });
+
+    test("should throw if glyph prop is not provided", () => {
+        expect(() => {
+            shallow(
+                <MSFTActionTrigger
+                    /* tslint:disable-next-line */
```

Comment:
Do you need all of the `tslint:disables`?

Reply:
Got rid of most